### DISCUSSION TITLE
Is there a way to highlight the last recently used tab?

### DISCUSSION THREAD

**Discussion by mauroporras** [kovidgoyal/kitty#7101] [Q&A] [answered]
I configured this map to quickly switch to the last recently used tab:

```
map cmd+l goto_tab -1
```

It'd be nice to have it highlighted, just like we can highlight the current tab, but with a different color.

For reference, this is what my config looks like, and as you can see, there's no way to identify the last recently used tab (AFAIK):

<img width="1511" alt="Screenshot 2024-02-05 at 09 38 29" src="https://github.com/kovidgoyal/kitty/assets/167700/fb651c1d-87d9-4dbe-a35b-f31c59f7a103">

Thank you very much for such an amazing project.

Edit: btw, thanks to Kitty I dropped tmux and my fingers are thanking me.

**Comment by kovidgoyal** [comment, marked as answer]
This information isnt available and generally speaking, the template
language isnt really designed for conditionals. Not something I am
personally interested in, but patches are welcome. Relevant code is in
tab_bar.py
